dom/dom_c.php

```diff
@@ -226,7 +230,7 @@ public function hasChildNodes(): bool {}
      * Indicates whether to copy all descendant nodes. This parameter is
      * defaulted to false.
      * </p>
-     * @return static The cloned node.
+     * @return static|false The cloned node.
```

Comment:
cf https://www.php.net/manual/en/domnode.clonenode.php#refsect1-domnode.clonenode-description
and https://github.com/php/php-src/blob/d7f9caf37c0915d1096c398671f4c92f0a11a1d3/ext/dom/php_dom.stub.php#L427